src/PrimitiveObsession.Test/EngineContainerBuilderTest.cs

```diff
@@ -47,7 +47,7 @@ public void should_resolve_TireCount()
         }
 
         [Fact]
-        public void should_TireCount_have_correct_value()
+        public void TireCount_should_have_correct_value()
         {

```

Comment:
Eh eh, non è che c'è una legge per cui i nomi dei metodi di test devono iniziare per should...
Anzi, la best practice è che i nomi dei metodi sian frasi di senso compiuto, corrette grammaticalmente :wink: